ClientKit/Display.cs

```diff
@@ -66,6 +68,26 @@ public override string ToString()
         }
     }
 
+    /// <summary>
+    /// Dimensions for a display input.
+    /// </summary>
+    public struct DisplayDimensions
+    {
+        public DisplayDimension Width { get; set; }
+        public DisplayDimension Height { get; set; }
+    }
+
+    /// <summary>
+    /// Projection clipping planes for a give viewer-eye-surface locus.

```

Comment:
Typo "give"->"given", and probably should be annotated that the only function that returns them returns the planes' locations at unit distance, so you'll usually want to divide by your "near" distance (but at that point you might as well just let us do the matrices for you...)